### PR TITLE
fixed issues with GPU and SpaCy

### PR DESCRIPTION
**Pull request by almarengo**
Added option in class Vectorizer to have tensor and model to GPU, if available
Fixed SpaCy package name when loading and deprecated error. Added exception with some instructions how to install "en_core_web_md" if package not found

### CODE REVIEW

File: sent2vec/splitter.py [hunk 1]
```diff
@@ -3,10 +3,17 @@
...
 import spacy
 
-os.environ['LANGUAGE_MODEL_SPACY'] = "en_core_web_md"
-nlp = spacy.load(os.environ['LANGUAGE_MODEL_SPACY'])
-sentencizer = nlp.create_pipe("sentencizer")
-nlp.add_pipe(sentencizer)
+# Make sure to have downloaded on your machine the package "en_core_web_sm" below 
+# In case run: python -m spacy download en_core_web_sm
+os.environ['LANGUAGE_MODEL_SPACY'] = "en_core_web_sm"
+#nlp = spacy.load(os.environ['LANGUAGE_MODEL_SPACY'])
```
Comment (pdrm83) [inline sent2vec/splitter.py:9]: You can delete line 9. Right?

**Comment by pdrm83** [comment]
Thank you. I don't have GPU on my laptop. Did you find a chance to test it as well?

**Comment by pdrm83** [comment]
BTW. If you believe this PR will resolve the issue raised by the community please kindly close the issue after this PR get merged.

File: sent2vec/splitter.py (hunk 1 above)
Comment (almarengo) [reply sent2vec/splitter.py:9]: Yes, line 9 can be deleted.

**Comment by almarengo** [comment]
I have a GPU on my laptop and I tested it. It works.

This PR will resolve two issues raised by the community. I will close those once this PR is merged.

**Review by pdrm83 — APPROVED** [review]

**Comment by almarengo** [comment]
Thanks for closing the PR. I just realized that I could have linked the issues with the PR :( Now I'm not able to close the issues but you should be. The solved issues are #8 and #10

**Pull request merged**